pkg/operator/controller/garden/garden/components.go

```diff
@@ -1112,6 +1112,7 @@ func (r *Reconciler) newPrometheus(log logr.Logger, garden *operatorv1alpha1.Gar
 			corev1.ResourceMemory: resource.MustParse("50G"),
 		},
 		AdditionalPodLabels: map[string]string{
+			v1beta1constants.LabelNetworkPolicyToPublicNetworks:                                                v1beta1constants.LabelNetworkPolicyAllowed,
```

Comment:
/hold

Does this need to be public networks or could it also be private networks, e.g. if Garden runtime cluster and seeds share the same virtual network?